python/paddle/fleet/meta_optimizers/common.py

```diff
@@ -47,10 +47,15 @@ def is_optimizer_op(op):
 
 
 class CollectiveHelper(object):
-    def __init__(self, role_maker, nrings=1, wait_port='6174'):
+    def __init__(self,
+                 role_maker,
+                 nrings=1,
+                 wait_port='6174',
+                 use_pipeline=False):
```

Comment:
Create a new `PipelineHelper` to do compile, which could inherited from `CollectiveHelper`. As `PipelineHelper` is specific for pipeline, it could be put into corresponding meta optimizer.

Reply:
Done. Create a new class PipelineHelper inherited from CollectiveHelper, and put it in the file defining pipeline_metat_optimizer.